### ISSUE TITLE
No intellisense for untyped **kwargs

### ISSUE THREAD

**Issue by russelldavis** [microsoft/pylance-release#119]
## Environment data
-   Language Server version: Pylance 2020.7.1
-   OS and version: macOS 10.15.5
-   Python version: 3.8

## Code Snippet
```python
def bug(**kwargs):
   # Reveals Dict[str, Unknown]
   reveal_type(kwargs)   
```

## Expected behaviour

When typing `kwargs.` inside the function, intellisense should pop up the list of members.

## Actual behaviour

Nothing happens.

## Logs
```
[Info  - 3:19:46 PM] Pylance language server 2020.7.1 starting
[Info  - 3:19:46 PM] Server root directory: /Users/russell/.vscode/extensions/ms-python.vscode-pylance-2020.7.1/server
[Info  - 3:19:46 PM] Loading configuration file at /Users/russell/dev/python/exek/pyrightconfig.json
[Info  - 3:19:46 PM] Assuming Python version 3.8
[Info  - 3:19:46 PM] Assuming Python platform Darwin
[Info  - 3:19:46 PM] No include entries specified; assuming /Users/russell/dev/python/exek/pyrightconfig.json
[Info  - 3:19:46 PM] Auto-excluding **/node_modules
[Info  - 3:19:46 PM] Auto-excluding **/__pycache__
[Info  - 3:19:46 PM] Auto-excluding .git
[Info  - 3:19:46 PM] Setting pythonPath for service "exek": "/Users/russell/dev/python/exek/.venv/bin/python"
[Info  - 3:19:46 PM] stubPath /Users/russell/dev/python/exek/typings is not a valid directory.
[Info  - 3:19:46 PM] Searching for source files
[Info  - 3:19:46 PM] Auto-excluding /Users/russell/dev/python/exek/.venv
[Info  - 3:19:46 PM] Found 5 source files
[Info  - 3:19:46 PM] Background analysis root directory: /Users/russell/.vscode/extensions/ms-python.vscode-pylance-2020.7.1/server
[Info  - 3:19:46 PM] Background analysis started
[BG] analyzing: /Users/russell/dev/python/exek/exek/foo.py ...
[BG]   parsing: /Users/russell/dev/python/exek/exek/foo.py (46ms)
[BG]   parsing: /Users/russell/.vscode/extensions/ms-python.vscode-pylance-2020.7.1/server/typeshed-fallback/stdlib/2and3/builtins.pyi (100ms)
[BG]   binding: /Users/russell/.vscode/extensions/ms-python.vscode-pylance-2020.7.1/server/typeshed-fallback/stdlib/2and3/builtins.pyi (48ms)
[BG]   binding: /Users/russell/dev/python/exek/exek/foo.py (0ms)
[BG]   checking: /Users/russell/dev/python/exek/exek/foo.py ...
[BG]     parsing: /Users/russell/.vscode/extensions/ms-python.vscode-pylance-2020.7.1/server/typeshed-fallback/stdlib/3/typing.pyi (19ms)
[BG]     binding: /Users/russell/.vscode/extensions/ms-python.vscode-pylance-2020.7.1/server/typeshed-fallback/stdlib/3/typing.pyi (16ms)
[BG]   checking: /Users/russell/dev/python/exek/exek/foo.py (58ms)
[BG] analyzing: /Users/russell/dev/python/exek/exek/foo.py (254ms)
```

**Comment by erictraut** [comment]
Can you elaborate what you mean by "list of member"? Are you referring to the members of the kwargs dictionary? The keys within this dictionary are unknown to a type checker. Where do you think it would get that information?

**Comment by russelldavis** [comment]
Ah sorry, no, I just mean the members of `Dict` itself, e.g., `keys`, `items`, `setdefault`, etc. The same behavior you'd see if you changed `def bug(**kwargs):` to `def bug(**kwargs: str):`.

**Comment by erictraut** [comment]
Ah, got it. Thanks for the additional explanation. We'll look into it.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version of Pylance/Pyright.

**Comment by russelldavis** [comment]
Wow, amazing turnaround time. You've been super helpful - thanks!

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.7.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202073-21-july-2020
